oeps/oep-0017-feature-toggles.rst

```diff
@@ -246,6 +247,14 @@ of the feature toggle and how long-lived it is expected to be.
 The diagram also labels which use cases are primarily driven by engineering
 teams (E) and/or business product teams (B).
 
+Note that a single feature toggle may transition through several use cases as a
+feature matures. An example feature toggle might first be used for beta testing a
+feature. While beta testing, the feature may mature from a beta test for
```

Comment:
Note: The example could easily be shortened by removing: `While beta testing, the feature may mature from a beta test for individual users to a beta test for several courses.` I personally like that this highlights the complexity involved within a use case, but I can also see an argument for less text.